Changelog.md

```diff
@@ -2,6 +2,119 @@
+    * [Pull request #409](https://github.com/gazebosim/gz-physics/pull/409)
+    * [Pull request #408](https://github.com/gazebosim/gz-physics/pull/408)
+    * [Pull request #407](https://github.com/gazebosim/gz-physics/pull/407)
+    * [Pull request #364](https://github.com/gazebosim/gz-physics/pull/364)
+    * [Pull request #362](https://github.com/gazebosim/gz-physics/pull/362)
+    * [Pull request #360](https://github.com/gazebosim/gz-physics/pull/360)
+    * [Pull request #363](https://github.com/gazebosim/gz-physics/pull/363)
+    * [Pull request #357](https://github.com/gazebosim/gz-physics/pull/357)
+
+1. Forward ports
+    * [Pull request #398](https://github.com/gazebosim/gz-physics/pull/398)
+    * [Pull request #5 ➡️  6 (main](https://github.com/gazebosim/gz-physics/pull/5 ➡️  6 (main)
+    * [Pull request #397](https://github.com/gazebosim/gz-physics/pull/397)
+    * [Pull request #2 ➡️  5](https://github.com/gazebosim/gz-physics/pull/2 ➡️  5)
+    * [Pull request #1 ➡️  2](https://github.com/gazebosim/gz-physics/pull/1 ➡️  2)
+    * [Pull request #5 -> 6](https://github.com/gazebosim/gz-physics/pull/5 -> 6)
+    * [Pull request #392](https://github.com/gazebosim/gz-physics/pull/392)
+    * [Pull request #2 ➡ 5](https://github.com/gazebosim/gz-physics/pull/2 ➡ 5)
+    * [Pull request #321](https://github.com/gazebosim/gz-physics/pull/321)
+    * [Pull request #5 ➡️  6 (main](https://github.com/gazebosim/gz-physics/pull/5 ➡️  6 (main)
+    * [Pull request #336 from ignitionrobotics/merge5main](https://github.com/gazebosim/gz-physics/pull/336 from ignitionrobotics/merge5main)
+    * [Pull request #Merge branch 'ign-physics5' into merge5main](https://github.com/gazebosim/gz-physics/pull/Merge branch 'ign-physics5' into merge5main)
+    * [Pull request #396 from gazebosim/chapulina/1_to_2](https://github.com/gazebosim/gz-physics/pull/396 from gazebosim/chapulina/1_to_2)
+    * [Pull request #391 from gazebosim/chapulina/5_to_6](https://github.com/gazebosim/gz-physics/pull/391 from gazebosim/chapulina/5_to_6)
+    * [Pull request #Merge branch 'main' into chapulina/5_to_6](https://github.com/gazebosim/gz-physics/pull/Merge branch 'main' into chapulina/5_to_6)
+
+1. dartsim: fix handling inertia matrix pose rotation
+    * [Pull request #351](https://github.com/gazebosim/gz-physics/pull/351)
+
+1. ABI tweaks to help bullet-featherstone
+    * [Pull request #395](https://github.com/gazebosim/gz-physics/pull/395)
+
+1. Add code coverage ignore file
+    * [Pull request #388](https://github.com/gazebosim/gz-physics/pull/388)
+
+1. Fix bullet classic entity management
+    * [Pull request #393](https://github.com/gazebosim/gz-physics/pull/393)
+
+1. Ignition to Gazebo transition
+    * [Pull request #386](https://github.com/gazebosim/gz-physics/pull/386)
+    * [Pull request #Fixed merge - ignition to gz](https://github.com/gazebosim/gz-physics/pull/Fixed merge - ignition to gz)
```

Comment:
This looks like a forward port, so we can drop it.

Reply:
Done